.ci/checker-framework.groovy

```diff
@@ -113,27 +113,43 @@ private static int checkCheckerFrameworkReport(final String profile) {
 private static List<List<String>> getCheckerFrameworkErrors(final String profile) {
     final List<String> checkerFrameworkLines = new ArrayList<>()
     final String command = "mvn -e --no-transfer-progress clean compile -P${profile}"
-    final Process process = getOsSpecificCmd(command).execute()
-    process.in.eachLine { final line ->
-        checkerFrameworkLines.add(line)
-        println(line)
+    final ProcessBuilder processBuilder = new ProcessBuilder(getOsSpecificCmd(command).split(' '))
+    processBuilder.redirectErrorStream(true)
+    final Process process = processBuilder.start()
+
+    final BufferedReader reader = new BufferedReader(new InputStreamReader(process.inputStream))
+    String lineFromReader
+    while ((lineFromReader = reader.readLine()) != null) {
+        println(lineFromReader)
+        checkerFrameworkLines.add(lineFromReader)
     }
     process.waitFor()
+
+    for (int index = checkerFrameworkLines.size() - 1; index >= 0; --index) {
+        final String curLine = checkerFrameworkLines.get(index)
+        if (curLine.startsWith('[ERROR]') || curLine.startsWith('[INFO]')) {
+            checkerFrameworkLines.remove(index)
+        }
+        else {
+            checkerFrameworkLines.set(index, curLine.replace('[WARNING]', '').trim())
+        }
+    }
     final List<List<String>> checkerFrameworkErrors = new ArrayList<>()
     for (int index = 0; index < checkerFrameworkLines.size(); index++) {
         final String line = checkerFrameworkLines.get(index)
-        if (line.startsWith('[WARNING]')) {
+        final String fileMatchingPattern = '.*src[\\\\/].*\\.java.*'
```

Comment:
Why do we need dot and kleene star at the end of the regexp? Can we do: `'.*src[\\\\/].*\\.java$`?